woocommerce/woocommerce-gutenberg-products-block
# Load ErrorPlaceholder CSS only in the editor

Small PR that renames the `<ErrorPlaceholder />` CSS files from `style.scss` to `editor.scss`. That component is only rendered in the editor so we don't need to bundle those styles in the frontend stylesheet.

### How to test the changes in this Pull Request:

1. Create a post with an _All Reviews_ block, for example.
2. Modify [this line in `ProductReviews.php`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/src/RestApi/Controllers/ProductReviews.php#L63) so reviews endpoint returns an error:
```diff
-if ( 'edit' === $request['context'] ) {
+if ( true ) {
```
3. Reload the post editor tab and verify it renders the error with the correct styles:
![imatge](https://user-images.githubusercontent.com/3616980/70717135-0af76100-1cee-11ea-8374-afeff69132ec.png)
